extensions/sql-database-projects/src/test/deploy/deployService.test.ts

```diff
@@ -140,125 +139,17 @@ describe('deploy service', function (): void {
 		const shellExecutionHelper = TypeMoq.Mock.ofType(ShellExecutionHelper);
 		shellExecutionHelper.setup(x => x.runStreamedCommand(TypeMoq.It.isAny(),
 			undefined, TypeMoq.It.isAny(), TypeMoq.It.isAny())).returns(() => Promise.resolve('id'));
-		const deployService = new DeployService(testContext.azureSqlClient.object, testContext.outputChannel, shellExecutionHelper.object);
+		const connectionService = new ConnectionService(testContext.outputChannel);
 		let connectionStub = sandbox.stub(azdata.connection, 'connect');
 		connectionStub.onFirstCall().returns(Promise.resolve(mockFailedConnectionResult));
 		connectionStub.onSecondCall().returns(Promise.resolve(mockConnectionResult));
 		sandbox.stub(azdata.connection, 'getUriForConnection').returns(Promise.resolve('connection'));
 		sandbox.stub(azdata.tasks, 'startBackgroundOperation').callThrough();
 
-		let connection = await deployService.getConnection(localDbSettings, false, 'master');
+		let connection = await connectionService.getConnection(localDbSettings, false, 'master');
 		should(connection).equals('connection');
 	});
 
-	it('Should update app settings successfully', async function (): Promise<void> {
```

Comment:
are these tests not applicable anymore? It would be nice to also have a test for the firewall rule error and prompt

Reply:
I removed the code for updating the settings because there were not used so removed the tests too